trackio/README.md

```diff
@@ -0,0 +1,5 @@
+---
+sdk: gradio
+sdk_version: 5.30.0
```

Comment:
I think it'd be better to get the user's `gr.__version__` and use that here to ensure a consistent experience locally and on Spaces.

Reply:
updated